fenix/app/src/main/java/org/mozilla/fenix/translations/preferences/nevertranslatesite/PrefNeverSiteTranslate.kt

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.preferences.nevertranslatesite
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.role
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.list.TextListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Firefox Preference: Never Translate Site Fragment Compose View.
+ *
+ * @param prefNeverTranslateSiteList List of [PrefNeverTranslateSiteListItem]s to display.
+ * @param onItemClick Invoked when the user clicks on the a item from the list.
+ */
+@Composable
+fun PrefNeverSiteTranslate(
```

Comment:
Please ensure this patch matches the new suggested pattern based on the [other patch](https://github.com/mozilla-mobile/firefox-android/pull/4226#discussion_r1406422920) before landing!

Reply:
Done